### ISSUE TITLE
Regarding mapping of anusvara .

### ISSUE THREAD

**Issue by arushidogra** [libindic/Transliteration#9]
The anusvara is always mapped to म्  which is wrong as it is sometimes mapped to न् and sometimes to 
म्. 

Here the output should have been chandan.
![screenshot from 2016-04-05 16 33 52](https://cloud.githubusercontent.com/assets/9354396/14279938/65232f54-fb4d-11e5-9c6e-44db7af0c870.png)

Here the output should have been sambandh
![screenshot from 2016-04-05 16 35 26](https://cloud.githubusercontent.com/assets/9354396/14279939/654470d8-fb4d-11e5-9318-c97a19cde2f8.png)

**Comment by copyninja** [comment]
I agree there is variation in the pronunciation and definitely this is not easy to be represented in English. But if you have approach to deduce this programmatically feel free to propose a patch.

**Comment by stultus** [comment]
I think If the letter following anusvara is of the Alveolo-palatal ('ta' family) and Dental ('tha' family) consonant, we can use 'n' instead of 'm'.
